sk0-front/src/app/auth/store/auth.effects.ts

```diff
@@ -59,7 +61,10 @@ export class AuthEffects {
       ofType(registerStarted),
       mergeMap((action) =>
         this.authService.register(action).pipe(
-          map((response) => registerSuccess(response)),
+          exhaustMap(async (response) => {
+            await this.notifyWith.toastMessage("Sikeres regisztráció!");
+            return registerSuccess(response);
+          }),
```

Comment:
My gut tell me that we are increasing the complexity here. It is an upfront investment, but shouldn't we hook rather the notification message to a `registerSuccess` action as an effect?

Reply:
that is correct